### PR TITLE
Ch responsive design

### PR DESCRIPTION
**Pull request by corwinharrell**

### CODE REVIEW

**Comment by rachelcope** [comment]
Looks good :airplane: (to paris)

File: app/assets/stylesheets/_profile.scss [hunk 1]
```diff
@@ -51,11 +51,29 @@
 // Styles for Alternative Profile Layout
 
 .supplier-info {
-  @include span-columns(2.5 of 8);
-  margin-top: $base-spacing;
+  margin: $base-spacing 0 $base-spacing $base-spacing;
+  //text-align: center;
```
Comment (smharley) [inline app/assets/stylesheets/_profile.scss]: 🚯

File: app/assets/stylesheets/_profile.scss [hunk 2]
```diff
@@ -51,11 +51,29 @@
...
+
+    @include media($large-screen-up) {
+      margin-left: 0;
+      text-align: left;
+    }
+  }
 
   &__metadata {
     display: block;
+
```
Comment (smharley) [inline app/assets/stylesheets/_profile.scss]: 👾

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -18,7 +18,9 @@
 <body class="<%= body_class %>">
   <%= render "flashes" -%>
   <%= render "header" -%>
-  <%= yield %>
+  <div class="page-wrapper">
```
Comment (smharley) [inline app/views/layouts/application.html.erb]: sometimes I just use the `main` element for things like this

**Comment by smharley** [comment]
Lookin good

**Pull request merged**